### PR TITLE
[MAJOR] Deploying application to Heroku.

### LINKED ISSUE #171: Testing deployment with Heroku
Epic/User story #165

### PR DESCRIPTION
**Pull request by Manraj45**
closes #171

### CODE REVIEW

**Review by Mnx458 — APPROVED** [review]
Great but it looks like we are also using postgres for production. Are we only using postgres from now on?

**Comment by Manraj45** [comment]
> Great but it looks like we are also using postgres for production. Are we only using postgres from now on?

Yes, we are switching to Postgres. We are using the Postgres database addon in Heroku.

**Comment by Mnx458** [comment]
> > Great but it looks like we are also using postgres for production. Are we only using postgres from now on?
> 
> Yes, we are switching to Postgres. We are using the Postgres database addon in Heroku.

File: frontend/server.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const serveStatic = require('serve-static')
+const path = require('path')
+
+const app = express()
+
+app.disable("x-powered-by");
```
Comment (nasaku898) [inline frontend/server.js:7]: Great addition for security!

**Review by nasaku898 — APPROVED** [review]

File: frontend/server.js (hunk 1 above)
Comment (Manraj45) [reply frontend/server.js:7]: Thanks. Actually, SonarCloud gave me this recommendation.

**Pull request merged**